policyengine_us/variables/gov/hhs/chip/is_chip_eligible_pregnant.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class is_chip_eligible_pregnant(Variable):
```

Comment:
```suggestion
class is_chip_eligible_pregnant_person(Variable):
```

Reply:
not adjusted yet

Reply:
similar format for other variables the `_person` can help us distinguish person level variables in the trace